examples/smoketest.sh

```diff
@@ -26,7 +46,7 @@ main() {
 
   for os in "${OSES[@]}"; do
     for i in $(seq 4); do
-      scenario$i $os
+      try scenario$i $os
```

Comment:
There is no `catch`. (Pun certainly intended.) `try` isn't bash, it's just a function I defined above.